### PR TITLE
Add .gitignore file

### PR DESCRIPTION
**Pull request by nathan-may**
Prevent API secrets file from being included in the repo. Also make some other small tweaks.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+# PyCharm
+#  JetBrains specific template is maintainted in a separate JetBrains.gitignore that can
+#  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
+#  and can be added to the global gitignore or merged into this file.  For a more nuclear
+#  option (not recommended) you can uncomment the following to ignore the entire idea folder.
+#.idea/
+
+# Alpaca-momentum
+app/config/secrets.py
```
Comment (nathan-may) [inline .gitignore:155]: API keys will now live in a `secrets.py` file

Comment (nathan-may) [reply .gitignore:155]: @1smay3 to add this to the README

File: app/repository/alpaca/alpaca_service.py [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
+from alpaca_trade_api.common import URL
 from rx import Observable
 from abc import abstractmethod
-from app.config.config import paper_url, paper_key, paper_secret
+from app.config.secrets import paper_key_id, paper_key_secret
 from app.repository.alpaca.models.Instructions import Instruction
 
-api = tradeapi.REST(base_url=paper_url, key_id=paper_key, secret_key=paper_secret, api_version='v2')
+api = tradeapi.REST(
+    base_url=URL("https://paper-api.alpaca.markets"),
```
Comment (nathan-may) [inline app/repository/alpaca/alpaca_service.py:12]: Wrapping this as a tidy up as it was doing some magic before.

**Review by 1smay3 — APPROVED** [review]
LGTM! 💯

**Pull request merged**